### ISSUE TITLE
Release 0.5.4 is not statically linked

### ISSUE THREAD

**Issue by s-yagodkin** [jianyuan/terraform-provider-sentry#73]
Hello! I got problem with new release 0.5.4 because it is not statically linked, like a previous one, causing problems with Alpine Linux (musl based distro)

**Comment by s-yagodkin** [comment]
**Platform:** linux_amd64

0.5.3
```bash
file terraform-provider-sentry
terraform-provider-sentry: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), statically linked, Go BuildID=0hSkRDGVeyPF0rMzDhj0/7E-bU3V8djfhuM_nOuoY/vIEO073-inBNlWVhsAJS/y7htpwMBhLZ090mfDHq6, stripped
```

0.5.4
```bash
terraform-provider-sentry: ELF 64-bit LSB executable, x86-64, version 1 (SYSV), dynamically linked, interpreter /lib64/ld-linux-x86-64.so.2, Go BuildID=wPvDxnHxb1KIJYLNug9B/4wS4_Zz31hW1vGYhRBfP/xVEO2Z7cxWdcDgcCj2Wt/DdzlgqgRRGO3DcLgyG9f, stripped
```

**Comment by jianyuan** [comment]
This should be fixed in [v0.5.5](https://github.com/jianyuan/terraform-provider-sentry/releases/tag/v0.5.5). I followed the official guide to publishing providers. This package is also available from the [Terraform registry](https://registry.terraform.io/providers/jianyuan/sentry) now!
